hedera-node/src/main/java/com/hedera/services/ledger/HederaLedger.java

```diff
@@ -603,26 +613,15 @@ private void updateXfers(AccountID account, long amount, TransferList.Builder xf
 		return (senderId != null) ? nftTransfer.setSenderAccountID(senderId) : nftTransfer;
```

Comment:
nit: Actually, I think that we can remove the `(senderId != null)` check. We used the same builder when minting and now the minting was refactored to the new model architecture, so this method is used only for transfers.

Reply:
Removed `null` check.